test/login-user-test.ts

```diff
@@ -7,7 +7,7 @@ import { loginUserMutation } from "./helpers/test-queries";
 import axios from "axios";
 
 describe("Login authentication tests", function () {
-  it("Logged in successfully", async function () {
+  it("Logged in successfully with remember me checked (7d token duration)", async function () {
```

Comment:
```suggestion
  it("Logged in successfully with remember me checked (long token duration)", async function () {
```

Imaginando que esses 7d é um dado que possa ser revisto futuramente, é melhor não ter ele "hardcoded" por ai, mesmo que seja aenas informativo. Porque senão vc vai ter que ficar caçando tudo o que tem 7 dias e trocar depois. Dando um significado pra ele (expiração longa, por exemplo), vc pode trocar apenas o valor, e o restante do código permaneceria coerente.